### ISSUE TITLE
Show actual date instead of approximates in cards

### ISSUE THREAD

**Issue by armaccloud** [nextcloud/deck#2128]
**Is your feature request related to a problem? Please describe.**
I am experiencing two issues
1. The subject text + Date next to each other is giving formatting issues, as #1672 should at least partially solve
2. In my overview I can only see when something is planned (exactly) when I open each card individually due to the approximate due date

**Describe the solution you'd like**
In the Anroid Deck app, the date is shown as an actual date, in the following format: `Jan 24`

This would be a solution to both the above issues;
1. Less width is used as a result from which the text will also gain some more space.
2. People can see the actual dates from their overview.

This will also bring the Android and Web App more in line with each other.

The following screenshot shows the difference in which the two apps show the same board. In my opinion the Android App shows the view more tidy.

_Android App_
![image](https://user-images.githubusercontent.com/9272078/87724753-498d5680-c7bc-11ea-967a-45467da9048f.png)

_Web App_
![image](https://user-images.githubusercontent.com/9272078/87724855-6fb2f680-c7bc-11ea-961d-1da9b3f96c2d.png)

**Comment by stefan-niedermann** [comment]
No strong opinion here, just a few notes:

The Android app does also show relative timespans, but only for due dates which are pretty close to "today":

![grafik](https://user-images.githubusercontent.com/4741199/87760980-f8f41880-c810-11ea-9ab8-cca48002d364.png)

> Less width is used as a result from which the text will also gain some more space.

This highly depends on the language and can easily be the other way around.

> People can see the actual dates from their overview.

I think a combination of absolute / relative timespans might be a good solution in this case.
- Assuming we need a precision of 1 day in the overview
- Relative timespans are cognitively easier to read
  - Use them for a short period around "today"
  - Use absolute timespans for dates which are not close to "today"

**Comment by armaccloud** [comment]
Yes @stefan-niedermann , that sounds like an elegant solution, to have the relative timespans some days around today for the web app also.

**Less width**
My remark
> Less width is used as a result from which the text will also gain some more space.

was pointed to using the format like the Android app shows the date, which is truncated like `Jan 24` instead of `January 24`. That way, you can also assure that the width of the absolute dates is mostly the same (maybe also using monospace to increase this?) making the view look consistent.

**Actual dates**
To solve the issue of not seeing the actual date, which in many cases is useful information (when planning dates & checking wehether all are correct) an idea would be to include a mouse-over on the timespan, so it shows the actual date when pointing the mouse over a relative timespan.

**Comment by kbeerepoot** [comment]
> Yes @stefan-niedermann , that sounds like an elegant solution, to have the relative timespans some days around today for the web app also.

I agree, and would think that one week is about the right point to make the difference

> > Less width is used as a result from which the text will also gain some more space.
> 
> was pointing to using the format like the Android app shows the date, which is truncated like `Jan 24` instead of `January 24`. That way, you can also assure that the width of the absolute dates is mostly the same (maybe also using monospace?) making the view look consistent.

I agree

> **Actual dates**
> To solve the issue of not seeing the actual date, which in many cases is useful information (when planning dates & checking wehether all are correct) an idea would be to include a mouse-over on the timespan, so it shows the actual date when pointing the mouse over a relative timespan.

I agree

**Comment by Hanskes** [comment]
Some progress? As you can see in the example at the top, there is hardly any space left for the title in Dutch.

**Comment by alimmroth** [comment]
Our customers have also requested to see actual dates. I think it would be very helpful to have a toggle in the settings for this behaviour.

**Comment by markusbegerow** [comment]
This idea would be great to see dates instead of relative time timespans
